index.html

```diff
@@ -17,7 +17,8 @@
       // editors, add as many as you like
       // only "name" is required
       editors:  [
-        { name: "Cyril Concolato", mailto: "cconcolato@netflix.com", url: "", company: "Netflix", companyURL: "https://www.netflix.com/"},
+        { name: "Cyril Concolato", w3cid: "36324", mailto: "cconcolato@netflix.com", url: "", company: "Netflix", companyURL: "https://www.netflix.com/"},
```

Comment:
Where do you find the `w3cid`?

Reply:
I think I can get it from the member-only view of TTWG participants. Pretty sure using it means that the W3C tooling picks us up as Editors and sends us relevant info on the spec-prod distribution list (not a busy traffic list, this is a Good thing).

Reply:
go to profile page via https://www.w3.org/users/myprofile/, and you will be redirected to URL with w3cid (or ask me anytime, of course)